packages/lesswrong/lib/ckEditorUtils.ts

```diff
@@ -19,6 +31,8 @@ export function generateTokenRequest(collectionName: CollectionNameString, field
           return reject(new Error('Cannot download a new token!'));
         }
 
+        cache.set(cacheKey, xhrResponse);
```

Comment:
Is this specifically related to the quick takes changes or was this a problem before?